JWPlayerPlugin/JWPlayerViewController.m

```diff
@@ -421,4 +414,33 @@ -(void)onPause:(JWEvent<JWStateChangeEvent> *)event {
                                                                  parameters:self.extensionsDictionary];
 }
 
+- (void)onFullscreen:(JWEvent<JWFullscreenEvent> *)event {
+    [self.closeButton removeFromSuperview];
+
+    if (event.fullscreen) {
+        self.player.forceFullScreenOnLandscape = YES;
+
+        if ([[UIDevice currentDevice]orientation] == UIInterfaceOrientationPortrait){
+            NSNumber *value = [NSNumber numberWithInt:UIInterfaceOrientationLandscapeLeft];
+            [[UIDevice currentDevice] setValue:value forKey:@"orientation"];
+        }
+        // A delay is needed since the video player in full screen mode deletes all subviews.
+        double delay = 0.1;
+        dispatch_time_t dispatchelay = dispatch_time(DISPATCH_TIME_NOW, delay * NSEC_PER_SEC);
+        dispatch_after(dispatchelay, dispatch_get_main_queue(), ^(void){
+            UIWindow *topWindow = [[[UIApplication sharedApplication].windows sortedArrayUsingComparator:^NSComparisonResult(UIWindow *win1, UIWindow *win2) {
+                return win1.windowLevel - win2.windowLevel;
+            }] lastObject];
+            UIView *topView = [[topWindow subviews] lastObject];
+            [topView addSubview:self.closeButton];
+        });
```

Comment:
There is no need to go through all created UIWindows and search one we need. You can take current key window and add close button. So it can be replaces with something like 
```
UIWindow *window = [UIApplication sharedApplication].keyWindow;
[window addSubview:self.closeButton]; 
```

Reply:
done

Reply:
Remove delay please

Reply:
done